doctr/utils/geometry.py

```diff
@@ -143,10 +143,40 @@ def rotate_abs_boxes(boxes: np.ndarray, angle: float, img_shape: Tuple[int, int]
     return rotated_boxes
 
 
+def remap_boxes(boxes: np.ndarray, orig_shape: Tuple[int, int], dest_shape: Tuple[int, int]) -> np.ndarray:
+    """ Remaps a batch of RotatedBbox (x, y, w, h, alpha) expressed for an origin_shape to a destination_shape,
+    This does not impact the absolute shape of the boxes
```

Comment:
Could you specify the description please? Remap from origin shape to destination shape, but for which operation (resize / rotation) ?